frontend/src/components/InfoDialog.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <v-container>
+    <info-markdown />
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import InfoMarkdown from "@/components/InfoMarkdown.md";
+</script>
+
+<style scoped>
+::v-deep.markdown-body p {
```

Comment:
There is a warning:

```
[@vue/compiler-sfc] ::v-deep usage as a combinator has been deprecated. Use :deep(<inner-selector>) instead.
```

See https://stackoverflow.com/questions/63986278/vue-3-v-deep-usage-as-a-combinator-has-been-deprecated-use-v-deepinner-se